presentation/src/main/java/org/sagebionetworks/research/presentation/perform_task/PerformTaskViewModel.java

```diff
@@ -357,6 +368,15 @@ protected void updateCurrentStep(@Nullable Step nextStep,
             if (stepView == null) {
                 LOGGER.warn("Step not found");
             }
+            //Initialize StepViewModelHelper for active tasks
```

Comment:
Do you think it'd be helpful to remove the previous step's ShowActiveUiStepViewModelHelper?

Reply:
Not sure it matters much.